CubeModLoader/DLL.h

```diff
@@ -17,6 +17,7 @@ class DLL
         // Callbacks
         FARPROC HandleChat;
         FARPROC HandleP2PRequest;
+        FARPROC HandleInventoryCheck;

```

Comment:
This name isn't consistent with "CheckInventoryFullHandler". Is this supposed to be "HandleCheckInventoryFull"?